installer_scripts/install-yb-voyager-debezium

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env bash
+
+export LC_CTYPE=en_US.UTF-8
+export LC_ALL=en_US.UTF-8
+
+set -e
+trap on_exit EXIT
+LOG_FILE=/tmp/install-yb-voyager-debezium.log
+YUM="sudo yum install -y -q"
+
+DEBEZIUM_VERSION="2.2.0-SNAPSHOT"
+DEBEZIUM_COMMIT_HASH="dbf21395320fe5da611281042667c32f36142812"
+
+on_exit() {
+	rc=$?
+	set +x
+	if [ $rc -eq 0 ]
+	then
+		echo "Done!"
+	else
+		echo "Script failed. Check log file ${LOG_FILE} ."
+		if [[ ${ON_INSTALLER_ERROR_OUTPUT_LOG:-N} = "Y" ]]
+		then
+			sudo cat ${LOG_FILE}
+		fi
+	fi
+}
+
+# Output a line to both STDOUT and STDERR. Use this function to output something to
+# user as well as in the log file.
+output() {
+	set +x
+	echo $1
+	>&2 echo $1
+	set -x
+}
+
+#=============================================================================
+# MAIN
+#=============================================================================
+main() {
+	set -x
+
+	os=$(uname -s)
+	case ${os} in
+		"Darwin")
+			install_core_utils_macos
+			exit $?
+			;;
+		"Linux")
+			# Proceed to distribution check.
+			;;
+		*)
+			echo "ERROR: unsupported os ${os}"
+			exit 1
+			;;
+	esac
+
+	# Linux
+	dist=$(cat /etc/os-release | grep -w NAME | awk -F'=' '{print $2}' | tr -d '"')
+	case ${dist} in
+		"CentOS Linux")
+			install_core_utils_centos
+			;;
+		"AlmaLinux")
+			install_core_utils_centos
+			;;
+		"Red Hat Enterprise Linux")
+			install_core_utils_centos
+			;;
+		"Red Hat Enterprise Linux Server")
+			install_core_utils_centos
+			;;
+		"Ubuntu")
+			install_core_utils_ubuntu
+			;;
+		*)
+			echo "ERROR: unsupported linux distribution: ${dist}"
+			exit 1
+			;;
+	esac
+
+	check_java
+	check_install_maven
+	install_debezium_server
+	output "Installation complete."
+}
+
+#=============================================================================
+# COMMON
+#=============================================================================
+
+check_binutils_version() {
+	output "Checking binutils version."
+	min_required_version='2.25'
+
+	# Example output of "ld -v" on CentOS/RHEL:
+	# GNU ld version 2.30-113.el8
+	# Example output of "ld -v" on Ubuntu:
+	# GNU ld (GNU Binutils for Ubuntu) 2.38
+	version=$(ld -v | awk '{print $NF}' | awk -F '-' '{print $1}')
+
+	if [[ ! "$version" > "$min_required_version" ]] 
+	then
+		echo "ERROR: unsupported binutils version ${version}. Update to binutils version > ${min_required_version} ."
+		exit 1
+	fi
+}
+
+centos_check_base_repo_enabled() {
+
+	output "Checking if base repo is enabled."
+
+	if [[ -z "$(yum repolist enabled | grep base)" ]]
+	then 
+		echo "ERROR: base repo is not enabled. Enable the repo and retry."
```

Comment:
What "repo" is this message talking about?

Reply:
Copied this prerequisite function from the existing `install-yb-voyager` script. I believe the base repo would contain the base utils without with the yum installs would fail. @ShivanshGahlot can you confirm?